admin/securedrop_admin/__init__.py

```diff
@@ -51,6 +51,13 @@ def validate(self, document):
             raise ValidationError(
                 message="Must not be an empty string")
 
+    class ValidateTime(Validator):
```

Comment:
You need a test for this class. See `def test_validate_ossec_username(self):` for an example.

Reply:
Done :+1: